cluster/gce/gci/configure.sh

```diff
@@ -131,8 +131,17 @@ function install-gci-mounter-tools {
 
 # Install node problem detector binary.
 function install-node-problem-detector {
-  local -r npd_version="v0.3.0"
-  local -r npd_sha1="2e6423c5798e14464271d9c944e56a637ee5a4bc"
+  if [[ -n "${NPD_VERSION:-}" ]]; then
+      local -r npd_version="${NPD_VERSION}"
+      if [[ -n "${NPD_TAR_HASH:-}" ]]; then
```

Comment:
I wasn't sure what "${NPD_TAR_HASH}" will set (null or empty string) when env variable is not set. I'll try this.